deploy/scripts/install.sh

```diff
@@ -48,15 +49,33 @@ az feature register --namespace "Microsoft.ContainerService" --name "EnableWorkl
 # Captures the Azure cloud endpoints/suffixes
 az cloud show -o json > $PRJ_ROOT/deploy/cloud_endpoints.json
 
+AKS_VERSION=$(az aks get-versions -l $LOCATION | jq -r '.orchestrators[].orchestratorVersion' | sort -rV | head -n1)
+
+if [ "${AKS_VERSION}" != "${MIN_AKS_VERSION_REQUIRED}" ]
```

Comment:
AKS has an ["N-2" version policy](https://learn.microsoft.com/en-us/azure/architecture/operator-guides/aks/aks-upgrade-practices) and the [latest version](https://kubernetes.io/docs/home/supported-doc-versions/#version-latest) is 1.26, which means no Azure cluster should have 1.22 available and this if check will never succeed. It should also be impossible for a cluster to be created with a version older than 1.22, which means this whole block is probably unnecessary.

Reply:
Added the suggested change to the latest commit.